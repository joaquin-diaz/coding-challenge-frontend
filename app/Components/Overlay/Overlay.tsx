import styled from '@emotion/styled';

const Overlay = styled.div`
  background-color: #eee;
  opacity: 0.8;
  position: absolute;
  height: 100vh;
  width: 100%;
  top: 0;
  right: 0;
  z-index: 20;
`;

export default Overlay;
