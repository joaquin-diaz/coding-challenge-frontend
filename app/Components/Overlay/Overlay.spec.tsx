import * as React from 'react';

import Overlay from './Overlay';
import { mount } from 'enzyme';

describe('<Overlay />', () => {
  it('should render', () => {
    const wrapper = mount(<Overlay />);
    expect(wrapper).toMatchSnapshot();
  });
});
