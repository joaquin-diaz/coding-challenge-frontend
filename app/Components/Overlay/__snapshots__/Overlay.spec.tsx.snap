// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Overlay /> should render 1`] = `
.emotion-0 {
  background-color: #eee;
  opacity: 0.8;
  position: absolute;
  height: 100vh;
  width: 100%;
  top: 0;
  right: 0;
  z-index: 20;
}

<Styled(div)>
  <div
    className="emotion-0"
  />
</Styled(div)>
`;
